### DISCUSSION TITLE
Masque Hiding Buttons

### DISCUSSION THREAD

**Discussion by Catchyst** [SFX-WoW/Masque#165] [Issues] [answered]
Hi guys,

I love this addon and I would kill to make it work, but it seems to have some kind of malfunction together with ElvUI.

If i enable Masque alone (without any style modules) some of my icons (on the actionbar) become invisible...

I'd appreciate any kind of answer, thanks in advance!

**Comment by StormFX** [comment]
If your icons are being hidden, this is a conflict/issue on the part of ElvUI as its it's the responsibility of authors to make their add-ons compatible with Masque, not the other way around. So you'll need to report the issue to the author of ElvUI.

**Comment by Blazeflack** [comment]
I'm not currently seeing any issues between Masque and ElvUI.

**Comment by PaniniGelato** [comment]
Is that ElvUI using Bartender4 or other action bar addons?
Action bar addons( like Bartender4) can use a set of custom configurations, depends on your settings
When u enable Masque alone, u will back to the Blizzard action bar with the origin configs, they are completely different bars

**Comment by StormFX** [comment, marked as answer]
This is an issue with ElvUI.
